lib/ansiblelint/skip_utils.py

```diff
@@ -72,10 +73,22 @@ def append_skipped_rules(pyyaml_data: str, file_text: str, file_type: FileType):
     return yaml_skip
 
 
+@lru_cache(maxsize=128)
```

Comment:
Why 128? Do we really need a default? I know for sure many repos with more than 128 yaml files. Also, I doubt we would run out of memory without a cache.

Make it 1024, but put a comment before to explaining that default is 128, which may not be big enough for all known use cases.

Reply:
This should be around number of total rules; Consider the following: this function is called from either `matchyaml` or `matchtasks` in `AnsibleLintRule`, which called basically in the following:

```python
for playbook in playbooks:
    for rule in rules:
        rule.run(...)
```

Basically LRU cache should live only for each iteration of inner loop (e.g after inner loop completed, we don't care anymore about content of `playbook`; 
Are my assumptions wrong?